### DISCUSSION TITLE
Setting for Quit On Close

### DISCUSSION THREAD

**Discussion by RyloRiz** [spacedriveapp/spacedrive#2325] [Feature Request]
When all windows are closed, there should be a setting to quit the app entirely instead of keeping it open (as it takes up space in the Dock if unpinned).

**Comment by utkubakir** [comment]
To give you some idea, Spacedrive supposed to run in the background so it can reindex/cleanup/process your files through custom jobs/actions etc for several workflows. They are all still work in progress, once we get closer we will share more information...

P.S. We will add an option to directly quit the app too.

**Reply by RyloRiz** [reply]
Alright, thanks for clearing that up. It'd be helpful if that ran as a background task instead of on the main app's thread to remove the icon from the Dock when not in use.
